cli/src/validate_command.rs

```diff
@@ -75,20 +75,26 @@ pub async fn run_validate(validate_args: ValidateArgs) -> anyhow::Result<i32> {
         }
         None => junit_paths
             .into_iter()
-            .map(JunitReportFileWithStatus::from)
+            .map(JunitReportFileWithTestRunnerReport::from)
             .collect(),
     };
     validate(junit_file_paths, codeowners_path).await
 }
 
-type JunitFileToReportAndParseIssues =
-    BTreeMap<String, (anyhow::Result<Option<Report>>, Vec<JunitParseIssue>)>;
-type JunitFileToReport = BTreeMap<String, Report>;
+type JunitFileToReportAndParseIssues = BTreeMap<
+    String,
+    (
+        anyhow::Result<Option<Report>>,
+        Vec<JunitParseIssue>,
+        Option<FileSetTestRunnerReport>,
+    ),
+>;
+type JunitFileToReport = BTreeMap<String, (Report, Option<FileSetTestRunnerReport>)>;
```

Comment:
Everything in this file should just be plumbing to pass around the test runner report